### PR TITLE
Add good background theme to member page

### PR DESCRIPTION
**Pull request by HelloYeew**
I think this page is about theme and helping each other so this can make user feel good when get in this page.

### CODE REVIEW

File: beatmap_collections/templates/beatmap_collections/team.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 {% block content %}
 <main class="text-light">
-    <div class="px-4 py-4 my-4 align-middle hero" style="background-image: linear-gradient(to bottom, transparent, var(--bg-color)), url({% static "img/team/hero-team.jpg" %}); background-position: top 20% center;">
+    <div class="px-4 py-4 my-4 align-middle hero" id="hero-sus" style="background-image: linear-gradient(to bottom, transparent, var(--bg-color)), url({% static "img/team/hero-team.jpg" %}); background-position: top 20% center;">
```
Comment (SainTurDaY27) [inline beatmap_collections/templates/beatmap_collections/team.html:31]: Name of id seems to be SUS. But It's alright.

**Review by Siraphop4Nene — APPROVED** [review]

**Review by SainTurDaY27 — APPROVED** [review]

File: beatmap_collections/templates/beatmap_collections/team.html [hunk 2]
```diff
@@ -170,7 +170,19 @@
             </div>
         </div>
     </section>
+    <embed src="{% static "among-us.mp3" %}" loop="true" autostart="true" width="2" height="0" id="player-sus">
```
Comment (SainTurDaY27) [inline beatmap_collections/templates/beatmap_collections/team.html:173]: Nice static with looping mp3, SUS.

**Review by SainTurDaY27 — APPROVED** [review]
SUS background, SUS theme, SUS member page.
Now everything is SUS!
Good!

**Review by some-random-idiot — APPROVED** [review]
Music to my ears!

**Comment by HelloYeew** [comment]
YES! LET"S MERGE THIS SUS!

**Pull request merged**